src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -498,37 +518,36 @@ ServerAddressList GrpcLb::Serverlist::GetServerAddressList(
     grpc_resolved_address addr;
     ParseServer(server, &addr);
     // LB token processing.
-    grpc_mdelem lb_token;
+    char lb_token[GPR_ARRAY_SIZE(server->load_balance_token) + 1];
     if (server->has_load_balance_token) {
       const size_t lb_token_max_length =
           GPR_ARRAY_SIZE(server->load_balance_token);
       const size_t lb_token_length =
           strnlen(server->load_balance_token, lb_token_max_length);
-      grpc_slice lb_token_mdstr = grpc_slice_from_copied_buffer(
-          server->load_balance_token, lb_token_length);
-      lb_token = grpc_mdelem_from_slices(GRPC_MDSTR_LB_TOKEN, lb_token_mdstr);
-      if (client_stats != nullptr) {
-        GPR_ASSERT(grpc_mdelem_set_user_data(
-                       lb_token, GrpcLbClientStats::Destroy,
-                       client_stats->Ref().release()) == client_stats);
-      }
+      memcpy(lb_token, server->load_balance_token, lb_token_length);
+      lb_token[GPR_ARRAY_SIZE(server->load_balance_token)] = '\0';
```

Comment:
should this be `lb_token_length` instead?

Reply:
Good catch.  Done.

Reply:
Thank you.